phpspec.yml.ci

```diff
@@ -0,0 +1,10 @@
+suites:
+  data_import_suite:
+    namespace: Ddeboer\DataImport
+    psr4_prefix: Ddeboer\DataImport
+extensions:
+  - PhpSpec\Extension\CodeCoverageExtension
+formatter.name: pretty
+code_coverage:
```

Comment:
Is it really needed to generate code coverage? PHPUnit generate already code coverage so i suppose this isn't really needed?